mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_qm_authority_update_job_profile.sql

```diff
@@ -0,0 +1,814 @@
+INSERT INTO ${myuniversity}_${mymodule}.job_profiles (id, jsonb) values
+('c7fcbc40-c4c0-411d-b569-1fc6bc142a92', '{
+	"id": "c7fcbc40-c4c0-411d-b569-1fc6bc142a92",
+	"name": "quickMARC - Default Update authority",
+	"description": "This job profile is used by the quickMARC to allow a user to update an SRS MARC authority record and corresponding Inventory authority. Profile cannot be edited or deleted",
+	"deleted": false,
+	"dataType": "MARC",
+	"metadata": {
+		"createdDate": "2022-04-19T09:07:47.667",
+		"updatedDate": "2022-04-19T09:09:10.382+0000",
+		"createdByUserId": "6a010e5b-5421-5b1c-9b52-568b37038575",
+		"updatedByUserId": "6a010e5b-5421-5b1c-9b52-568b37038575"
+	},
+	"userInfo": {
+		"lastName": "System",
+		"userName": "System",
+		"firstName": "System"
+	},
+	"childProfiles": [],
+	"parentProfiles": []
+}
+') ON CONFLICT DO NOTHING;
+
+INSERT INTO ${myuniversity}_${mymodule}.action_profiles (id, jsonb) values
+('f0f788c8-2e65-4e3a-9247-e9444eeb7d70',
+'{
+	"id": "f0f788c8-2e65-4e3a-9247-e9444eeb7d70",
+	"name": "quickMARC - Default Update MARC authority",
+	"action": "UPDATE",
+	"deleted": false,
+	"metadata": {
+		"createdDate": "2020-11-30T09:02:39.96",
+		"updatedDate": "2020-11-30T11:57:24.083+0000",
+		"createdByUserId": "6a010e5b-5421-5b1c-9b52-568b37038575",
+		"updatedByUserId": "6a010e5b-5421-5b1c-9b52-568b37038575"
+	},
+	"userInfo": {
+		"lastName": "System",
+		"userName": "System",
+		"firstName": "System"
+	},
+	"description": "Updates existing SRS MARC authority record",
+	"folioRecord": "MARC_BIBLIOGRAPHIC",
+	"childProfiles": [{
+			"id": "041f8ff9-9d17-4436-b305-1033e0879501",
+			"order": 0,
+			"content": {
+				"id": "041f8ff9-9d17-4436-b305-1033e0879501",
+				"name": "quickMARC - Default Update MARC authority",
+				"deleted": false,
+				"metadata": {
+					"createdDate": 1606726926555,
+					"updatedDate": 1606726926555,
+					"createdByUserId": "6a010e5b-5421-5b1c-9b52-568b37038575",
+					"updatedByUserId": "6a010e5b-5421-5b1c-9b52-568b37038575"
+				},
+				"userInfo": {
+					"lastName": "System",
+					"userName": "System",
+					"firstName": "System"
+				},
+				"description": "Updates existing SRS MARC authority record",
+				"childProfiles": [],
+				"mappingDetails": {
+					"name": "marcBib",
+					"recordType": "MARC_BIBLIOGRAPHIC",
+					"mappingFields": [],
+					"marcMappingOption": "UPDATE",
+					"marcMappingDetails": []
+				},
+				"parentProfiles": [],
+				"existingRecordType": "MARC_BIBLIOGRAPHIC",
+				"incomingRecordType": "MARC_BIBLIOGRAPHIC",
+				"marcFieldProtectionSettings": []
+			},
+			"contentType": "MAPPING_PROFILE",
+			"childSnapshotWrappers": []
+		}
+	],
+	"parentProfiles": [{
```

Comment:
There is no need to put parent and child profiles. Please remove them from all files.